modules/graphql_content/src/Plugin/GraphQL/Enums/Languages.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Drupal\graphql_content\Plugin\GraphQL\Enums;
+
+use Drupal\Core\Language\LanguageManagerInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\graphql_core\GraphQL\EnumPluginBase;
+use Drupal\graphql_core\GraphQLSchemaManagerInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Generates an enumeration of numbers.
+ *
+ * @GraphQLEnum(
+ *   id = "language",
+ *   name = "Language"
+ * )
+ */
+class Languages extends EnumPluginBase implements ContainerFactoryPluginInterface {
+
+  /**
+   * The language manager.
+   *
+   * @var \Drupal\Core\Language\LanguageManagerInterface
+   */
+  protected $languageManager;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function __construct(array $configuration, $pluginId, $pluginDefinition, LanguageManagerInterface $languageManager) {
+    parent::__construct($configuration, $pluginId, $pluginDefinition);
+
+    $this->languageManager = $languageManager;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
+    return new static(
+      $configuration,
+      $plugin_id,
+      $plugin_definition,
+      $container->get('language_manager')
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildValues(GraphQLSchemaManagerInterface $schemaManager) {
+    $values = [];
+
+    foreach ($this->languageManager->getLanguages() as $language) {
+      $values[] = [
```

Comment:
This is not how you build an enum. An enum is just a key=>value pair where the value is just an assoc array with a 'name' and a 'value' entry.

e.g. like so:

```php
$values = [
  ['name' => 'English', 'value' => 'en'],
  ['name' => 'German', 'value' => 'de'],
];
```